File: Project 1/Project1/Project1/Day.swift
//
//  Day.swift
//  Project1
//
//  Created by Kaylin Shioshita on 3/4/18.
//  Copyright © 2018 Kaylin Shioshita. All rights reserved.
//

import Foundation
import RealmSwift

class Day: Object{
    
    @objc dynamic var notes = ""
    var steps = List<String>()
    @objc dynamic var condition = 0
    @objc dynamic var daycode = ""
    @objc dynamic var timeofday = -1
//    var condition = String()
    
}

File: Project2/Project2Milestone2/app/src/main/java/kaylinshioshita/skincaremanager/Day.java
package kaylinshioshita.skincaremanager;

import io.realm.RealmList;
import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.Required;

/**
 * Created by kaylinshioshita on 4/23/18.
 */

public class Day extends RealmObject {
    @Required
    @PrimaryKey
    private String notes;
    private RealmList<String> steps;
    private int condition;
    private String daycode;
    private int timeOfDay;

    public Day() {
        timeOfDay = -1;
        daycode = "";
        condition = 0;
        notes = "";
    }

    public int getCondition() {
        return condition;
    }

    public int getTimeOfDay() {
        return timeOfDay;
    }
}
